src/components/ColorPicker.jsx

```diff
@@ -19,15 +18,18 @@ import * as ColorTool from '../helpers/colorTool';
 import uncontrolled from '../helpers/uncontrolled';
 import * as CommonTypes from '../helpers/commonTypes';
 import useTranslate from '../helpers/useTranslate';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
`@material-ui/core/styles` import should occur before import of `./ColorButton`

Reply:
Fixed